docs/source/simulation_study_results.rst

```diff
@@ -0,0 +1,317 @@
+| dataset  | Candidate publications | Relevant publications | Inclusion rate (%) |
++==========+========================+=======================+====================+
+| Nudging  | 1847                   | 100                   | 5.41               |
++----------+------------------------+-----------------------+--------------------+
+| PTSD     | 5031                   | 38                    | 0.76               |
++----------+------------------------+-----------------------+--------------------+
+| Software | 8896                   | 104                   | 1.17               |
++----------+------------------------+-----------------------+--------------------+
+| Ace      | 2235                   | 41                    | 1.83               |
++----------+------------------------+-----------------------+--------------------+
+| Virus    | 2304                   | 114                   | 4.95               |
++----------+------------------------+-----------------------+--------------------+
+| Wilson   | 2333                   | 23                    | 0.99               |
++----------+------------------------+-----------------------+--------------------+
+
+Models
+------
+
+To assess effect of different ASReview models, seven models were
+evaluated differing in terms of the classification technique (Naive
+Bayes, Linear Regression, Support Vector Machine, and Random Forest) and
+the feature extraction strategy they adopt (TF-IDF and Doc2vec). The
+Naive Bayes + TF-IDF model is the current default in ASReview. Note that
+the ASReview GUI currently only offers freedom of choice for a
+classification technique. However, the models presented here are not
+exhaustive as the ASReview backend implements various other
+configurations.
+
+Evaluation
+----------
+
+Model performance was assessed by two different measures, Work Saved
+over Sampling (WSS), and Relevant References Found (RRF). WSS indicates the reduction in publications needed to be screened, at a
+given level of recall (Cohen et al. 2006). Typically measured at a
+recall level of 0.95 (Cohen et al. 2006), WSS95 yields an estimate of
+the amount of work that can be saved at the cost of failing to identify
+5% of relevant publications. In the current study, WSS is computed at
+0.95 recall. RRF statistics are computed at 10%, representing the
+proportion of relevant publications that are found after screening 10%
+of all publications.
+
+Furthermore, model performance was visualized by plotting recall curves.
+Plotting recall as a function of the proportion of screened publications
+offers insight in model performance throughout the entire screening
+process (Cormack and Grossman 2014; Yu, Kraft, and Menzies 2018). The
+curves give information in two directions. On the one hand they display
+the number of publications that need to be screened to achieve a certain
+level of recall (1-WSS), but on the other hand they present how many
```

Comment:
I'm not sure I understand this part. What is the 1-WSS?

Reply:
I see I was a bit unclear. I mean to explain that you can read the recall curve in terms of WSS or RRF. In case of the WSS, the x-axis of the recall curves actually show the proportion of papers that have been screened instead of the proportion of papers that have not been screened, hence the 1-WSS. 

> On the one hand they display the proportion of publications that need to be screened (1-WSS) to achieve a certain level of recall.

Reply:
Correct me if I'm wrong, but I think you're trying to convey that you could flip the x- and y-axes and learn something interesting, right? (i.e. either ask which % of papers need to be reviewed, or what % of inclusions we find.)

Maybe the easiest way to convey this is to explain this is to describe it visually? Otherwise it might be a little hard for the non-expert to read.

I still don't understand the 1-WSS though. That would be the percentage of papers read + % of inclusions not found?

Reply:
Yep that's waht I'm trying to convey! Then how about:

> The x-axis represents the proportion of screened publications, the y-axis represents the proportion of relevant publications found. The curves give information in two directions. On the one hand, they display the proportion of relevant publications found for any point during the screening process (on the x-axis). On the other hand, they present the proportion of publications that need to be screened to achieve a certain level of recall (on the y-axis). The quicker the curve reaches towards the top of the y-axis, the better the performance. 

About the 1-WSS: correct me if I'm wrong, but I figured that WSS@95% is 60%, that means that after having seen 40% of publications you've found 95% of relevant publications. But I figure from your response that this is technically not 100% correct?

Reply:
Yes, I think that's much easier to understand.

No, as I understand WSS@95% == 60% means that after reading 35% of the publications we have found 95% of the publications (60 + 35 == 95). It is the amount work saved, instead of doing it randomly. So WSS@x for random search strategy should be 0 for any x. In your case if WSS@100% == 60% then you need to read 40%.

But honestly, it's quite confusing, so it wouldn't surprise me if I also got the numbers wrong somehow...

Reply:
We will close this discussion for now. Thanks for the discussion. We will address this in a new issue/PR.